GitToDoList/Screen/Login/LoginVM.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  LoginVM.swift
+//  GitToDoList
+//
+//  Created by 선민승 on 2021/03/20.
+//
+
+import SafariServices
+
+import Moya
+import RxCocoa
+import RxSwift
+
+class LoginViewModel {
+    let disposeBag = DisposeBag()
+    let emailTextPublishSubject = PublishSubject<String>()
+    let passwordTextPublishSubject = PublishSubject<String>()
```

Comment:
이거 없어질 것이긴 하지만! 나중엔 input이랑 output으로 나눠보면 좋을 것 같아요! 
맞는진 모르겠지만 이런식으로..? VM생성할 때 생성자에 둘을 PublishSubject로 묶어줄 수 있을 것 같아요.
```suggestion
    let editEmail: AnyObserver<Void>
    let editPasswrod: AnyObserver<Void>
    
    let email: Observable<String>
    let password: Observable<String>
```